libraries/chain/include/eosio/chain/account_object.hpp

```diff
@@ -104,8 +104,9 @@ CHAINBASE_SET_INDEX_TYPE(eosio::chain::account_object, eosio::chain::account_ind
 CHAINBASE_SET_INDEX_TYPE(eosio::chain::account_metadata_object, eosio::chain::account_metadata_index)
 CHAINBASE_SET_INDEX_TYPE(eosio::chain::account_ram_correction_object, eosio::chain::account_ram_correction_index)
 
-
+// @ignore id
```

Comment:
There were lots of id parameters added to the ignore lists.  I believe that id was there when this was originally put in place.  Is there a reason that it wasn't needed before and it is needed now?

Reply:
The ids were not present in any of the reflections where I put this.  I assumed the reflections were correct absent some other bug report.

Reply:
I went back And those were not there originally, also they were not reported when I ran the script and reported this initially, so I would like to figure out why it was added.

Reply:
@ndcgundlach still waiting on this